### DISCUSSION TITLE
答应我，下次别再-f了好吗

### DISCUSSION THREAD

**Discussion by shoucandanghehe** [26F-Studio/Techmino#253] [General]
@MrZ626 
![66DN7QN2KZ T 1S5FHT7%R0](https://user-images.githubusercontent.com/51957264/131230173-90fb0ddb-578a-451b-8e0f-c2214be98458.png)
![U03Z2CLVK0$P04}9Q28C10A](https://user-images.githubusercontent.com/51957264/131230174-e60f4e6a-234a-48c9-bc11-6c8cb2985ac8.jpg)

**Comment by C29H25N3O5** [comment]
<img width="332" alt="Screenshot 2021-11-30 at 14 56 38" src="https://user-images.githubusercontent.com/87485746/144126736-c7684390-d3d4-4df4-a0d9-61ea30cb8861.png">
